### DISCUSSION TITLE
Pre-existing worlds.

### DISCUSSION THREAD

**Discussion by xFusionLordx** [Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#13] [Q&A] [answered]
Are there any potential issues with adding this to an existing world? IE: Dungeon overlap?

**Comment by Digitalroot** [comment, marked as answer]
There is not, due to the way locations work in VH, there is a, one dungeon per zone limit. A zone is 64x64 units. Rooms are never placed outside a zone, thus no overlap.

**Comment by DustinWi** [comment]
Resurrecting the post.

How does this work with existing worlds?  At what point does a dungeon get generated?  If I'm already playing a world, which dungeons will get the new settings?

**Reply by DustinWi** [reply]
Never mind.  Found my answer here.
https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms/discussions/3f

I was too hasty to ask and not search.
